src/api.js

```diff
@@ -5,13 +5,29 @@ const newsApi = axios.create({
   baseURL: "https://emiles-nc-news.herokuapp.com/api",
 });
 
-export const getArticles = () => {
+export const getArticles = (req, res, next) => {
   return newsApi
     .get("/articles")
     .then(({ data }) => {
       return data.articles;
     })
-    .catch((err) => {
-      console.log(err);
-    });
+    .catch(next);
+};
+
+export const getArticlesByTopic = (topic_slug, res, next) => {
```

Comment:
if you pop in an nchelp tomorrow we'll walk you through using the axios `params` object and how it can help us to handle queries (there may even be a Q&A session on this very subject tomorrow morning)